packages/components/src/components/dropdown/dropdown.css

```diff
@@ -11,11 +11,10 @@
 
 scale-dropdown {
   --font-weight: var(--telekom-typography-font-weight-bold);
-  --height: var(--telekom-spacing-unit-x12);
-  --height-small: var(--telekom-spacing-unit-x10);
+  --height: var(--telekom-spacing-unit-x11);
   --spacing-x: var(--telekom-spacing-unit-x3);
-  --spacing-dropdown: var(--telekom-spacing-unit-x3)
-    var(--telekom-spacing-unit-x10) 0 calc(var(--spacing-x) - 1px);
+  --spacing-dropdown: 21px var(--telekom-spacing-unit-x10) 5px
```

Comment:
`21px` and `5px` are funny, is this like that in the design spec? in any case, this component will be deprecated imminently so no need to worry, I was just wondering…

Reply:
Unfortunately, yes, I had already talked to the designers, that partly the spacings are not included in the design tokens and that is a bit of a pity.